physlr/physlr.py

```diff
@@ -1332,6 +1332,79 @@ def determine_molecules_bc_cosine_of_squared(g, u):
                 for community in
                 Physlr.detect_communities_cosine_of_squared(g, bi_connected_component)]
 
+    @staticmethod
+    def partition_subgraph_into_bins_randomly(node_set, max_size=40):
```

Comment:
```suggestion
    def partition_subgraph_into_bins_randomly(node_set, max_size=50):
```
Has this value been optimized, and found 40 to be best? If it's arbitrary and unoptimized, I prefer values along the 1-2-5, scale.

Reply:
umm I tested different values for a few examples and something between 30-40 seems to perform better. The lower the faster, and it is more likely to overcome some misassemblies by over-splitting. But if we set it to really low values then community detection sucks.

Reply:
Can you compare 40 to 50? If 40 performs better, I'm fine with that value.

Reply:
I have done that for fly. It's faster and not worst in terms of results quality  (theoretically might be better, but have not seen that for fly). So now do you prefer 40 or 50? I can test for human as well.

Reply:
Do test zebrafish and human please.

Reply:
I just decided to go with 50 for now and postpone this tuning to later. The difference is not that much and it's one less thing to worry about now :smile: